test/version.jl

```diff
@@ -219,6 +219,12 @@ for major=0:3, minor=0:3, patch=0:3
     end
 end
 
+# banner
+import Base.banner
```

Comment:
This doesn't exist anymore on master

Reply:
Must have been a rebasing problem. Fixed, thanks1